public/app/features/explore/Logs/Logs.tsx

```diff
@@ -265,38 +265,37 @@ const UnthemedLogs: React.FunctionComponent<Props> = (props: Props) => {
     }
 
     // check if we have dataFrames that return the same level
-    const logLevelsArray: Array<{ levelStr: string; logLevel: LogLevel }> = [];
+    const logLevelsArray: LogLevel[] = [];
     logVolumeDataFrames.forEach((dataFrame) => {
       const { level } = getLogLevelInfo(dataFrame, logsVolumeData?.data ?? []);
-      logLevelsArray.push({ levelStr: level, logLevel: getLogLevel(level) });
```

Comment:
getLogLevel() was being incorrectly used. It worked by pure chance, as it's meant to get the log level from a log line (from the actual log line body).